Panama OCDID file update

MarcinMisztalTR opened the pull request:
Hello,
According to information included in:
https://en.wikipedia.org/wiki/2019_Panamanian_general_election
https://www.tribunal-electoral.gob.pa/eventos-electorales/plan-general-elecciones-plagel/
https://www.tribunal-electoral.gob.pa/eventos-electorales/plan-general-elecciones-plagel/cuadro-05-diputado-candidatos-3/

I've added provinces (https://en.wikipedia.org/wiki/Provinces_of_Panama) and electoral districts/election circuits.

Please let me know if more details/data are needed, thank you!

HKSenior commented on identifiers/country-pa/README:
```diff
@@ -0,0 +1,9 @@
+README
```
Lets keep the title consistent with other directories (example [Canada](https://github.com/opencivicdata/ocd-division-ids/blob/master/scripts/country-ca/README.md?plain=1#L1)).
```suggestion
# Open Civic Data Divisions: Panama
```

HKSenior commented on identifiers/country-pa/README:
```diff
@@ -0,0 +1,9 @@
+*   provinces.csv: contains the list of provinces in Panama [https://en.wikipedia.org/wiki/Provinces_of_Panama]
+*   electoral_districts.csv: contains list of electoral districts (39 circuits) in Panama
+
+OCD ID Types used used:
```
```suggestion
OCD ID Types used:
```

MarcinMisztalTR replied:
corrected

MarcinMisztalTR commented:
Thank you for the review, both issues in README has been corrected and committed to PR

jloutsenhizer commented on identifiers/country-pa/README:
```diff
@@ -0,0 +1,9 @@
+File contents in `country-pa` directory:
+*   top_level.csv: contains country level OCD-ID
+*   provinces.csv: contains the list of provinces in Panama [https://en.wikipedia.org/wiki/Provinces_of_Panama]
+*   electoral_districts.csv: contains list of electoral districts (39 circuits) in Panama
```
Is there an official source for these electoral districts that could be linked in the readme? It'd also help with checking the `electoral_districts.csv` file for correctness

MarcinMisztalTR replied:
We were using mainly this page: https://www.iso.org/obp/ui/#iso:code:3166:PA

Additional ones:
https://en.wikipedia.org/wiki/ISO_3166-2:PA
https://alertas.directoriolegislativo.org/wp-content/uploads/2019/05/ENG-Post-election-report-Panama.pdf

jloutsenhizer commented on identifiers/country-pa/electoral_districts.csv:
So my understanding from https://pdba.georgetown.edu/ElecSys/Panama/panama.html is that there are 26 single-member districts and 13 multi-member districts.  Do you think it might make sense to use different types to distinguish the two types?

jloutsenhizer replied:
It looks like there's a resource with the electoral results which also labels which districts are single-member and which are multimember: https://www.tribunal-electoral.gob.pa/eventos-electorales/plan-general-elecciones-plagel/cuadro-06-diputado-proclamados-6/

I'd recommend updating the electoral distirct types to distinguish between the "uninominal" and "plurinominal" districts (unless these change frequently).

I don't have a strong preference on what these types are, one option would be `cp` and `cu` for circuito plurinominal and circuito uninominal

MarcinMisztalTR replied:
Hello, thank you for the response!
Are they good examples?
ocd-division/country:pa/province:bocas_del_toro/cp:1    
ocd-division/country:pa/province:panama/cu:5

jloutsenhizer replied:
Sorry for the delay, yes those examples look good to me

MarcinMisztalTR commented on identifiers/country-pa/electoral_districts.csv:
```diff
@@ -0,0 +1,40 @@
+id,name
+ocd-division/country:pa/province:bocas_del_toro,Bocas del Toro
```
Missing /ed:1 added

MarcinMisztalTR commented:
Hello,
I've updated the files:
- changed indigenous regions type from 'province' to 'territory'
- changed ed to cp/cu (circuito plurinominal / circuito uninominal)
- changed 'Guna Yala' to 'Kuna Yala' (done according to document which came directly from our media contact at the Parliament and this is how they will be reporting the results in the next election)

Please let me know if more details or changes are needed. Thank you.

jpmckinney commented on identifiers/country-pa/provinces_and_indigenous_regions.csv:
```diff
@@ -0,0 +1,13 @@
+ocd-division/country:pa/province:panama,Panamá
+ocd-division/country:pa/province:panama_oeste,Panamá Oeste
+ocd-division/country:pa/province:veraguas,Veraguas
+ocd-division/country:pa/territory:kuna_yala,Kuna Yala
+ocd-division/country:pa/territory:ngabe-bugle,Ngäbe-Buglé
```
Aren't there 4 territories total?

jpmckinney replied:
Also, we would typically use the numbers for provinces and codes for territories from ISO 3166-2: https://en.wikipedia.org/wiki/ISO_3166-2:PA

MarcinMisztalTR replied:
Hello, are you able to provide some examples?
Do I understand this correctly that I should replace:
"ocd-division/country:pa/**province:bocas_del_toro**/cp:1,Bocas del Toro" by "ocd-division/country:pa/**province:1**/cp:1,Bocas del Toro"
and "ocd-division/country:pa/**territory:ngabe-bugle**/cu:1,Ngäbe-Buglé-1" by "ocd-division/country:pa/**territory:nb**/cu:1,Ngäbe-Buglé-1"

MarcinMisztalTR replied:
> Aren't there 4 territories total?

Yes, there are 4, but I've added only those which were attached in document from Government (for future election). I can add missing 2 (Emberá-Wounaan and Naso Tjër Di) if this is OK for you (both most likely won't have any ed for next election)

jloutsenhizer replied:
I think trying to be complete for the entity type being added would be iedal so adding the other territories would be ideal

jpmckinney reviewed (commented):
> changed 'Guna Yala' to 'Kuna Yala' (done according to document which came directly from our media contact at the Parliament and this is how they will be reporting the results in the next election)

But:

> The area was formerly known as San Blas, and later as Kuna Yala, but the name was changed in October 2011 to "Guna Yala" when the Government of Panama recognized the claim of the people that "Guna" was a closer representation of the name.

Same on the [Spanish Wikipedia page](https://es.wikipedia.org/wiki/Comarca_Guna_Yala):

> El área fue formalmente conocida como San Blas, junto con el nombre autóctono de Kuna Yala, pero se cambió en octubre de 2011 cuando el Gobierno de Panamá reconoció la afirmación del pueblo Guna que en su lengua materna no existía el [fonema](https://es.wikipedia.org/wiki/Fonema) para la letra "K", y que el nombre oficial debería ser "Guna Yala". Guna Yala en [lengua guna](https://es.wikipedia.org/wiki/Idioma_guna) significa "Tierra Guna" o "Montaña Guna".

So, I think it should be Guna Yala.

----

> I'd recommend updating the electoral distirct types to distinguish between the "uninominal" and "plurinominal" districts (unless these change frequently).

I think they should all be `ed`. If a single-member district changes to a multi-member district (e.g. without changing its boundary) then I would consider it the same electoral division.

MarcinMisztalTR commented:
> So, I think it should be Guna Yala.

Regarding Guna Yala/Kuna Yala, we have received this as a official name for this region:
<img width="146" alt="image" src="https://github.com/opencivicdata/ocd-division-ids/assets/97161725/edde68da-f9a1-4fa9-8637-1d61a1597bde">

MarcinMisztalTR commented:
> I think they should all be `ed`. If a single-member district changes to a multi-member district (e.g. without changing its boundary) then I would consider it the same electoral division.

Should I do it now or you need to discuss it with team?

jloutsenhizer commented:
RE `ed` vs `cu` and `cp`. I agree with @jpmckinney's assessment. On second look it does look like which ones are multimember can vary and the numbering stays consistent for the set of districts. So moving forward with `ed` sounds good to me

jpmckinney commented:
> Regarding Guna Yala/Kuna Yala, we have received this as a official name for this region:

I don't know why election results are reported using the old spelling, but it has officially been renamed with a G, and what we are modelling are official political divisions – we are not creating a model of how election results are reported. Most other official government sources use the G like https://www.mingob.gob.pa/gobernacion-la-comarca-guna-yala/

MarcinMisztalTR commented:
Hello,
I've:
- reverted 'Kuna Yala' to 'Guna Yala'
- renamed provinces to numbers and territories to codes [https://en.wikipedia.org/wiki/ISO_3166-2:PA] 
- added 2 missing territories

I'm not sure if numbers/codes is what you meant, please let me know if it's OK or should be changed in different way/pattern.

Thank you

MarcinMisztalTR commented:
Hello, I would like to ask if there is anything more to add in this ticket?
